docs/scaffolds/stomach.rst

```diff
@@ -0,0 +1,147 @@
+Stomach Scaffold
+=============
+
+The current stomach scaffold is ``3D Stomach 1`` built from ``class MeshType_3d_stomach1``.
+The human variant is shown in :numref:`fig-scaffoldmaker-human-stomach`.
+
+.. _fig-scaffoldmaker-human-stomach:
+
+.. figure:: ../_images/scaffoldmaker_human_stomach.jpg
+   :align: center
+
+   Human stomach scaffold.
+
+The stomach scaffold is a 3-D volumetric model of the stomach representing the fundus, corpus, cardia, antrum, pylorus,
+and a short distal section of the esophagus as well as a short proximal section of the duodenum.
+
+Variants
+--------
+
+The stomach scaffold is provided with parameter sets for the following four species, which differ in shape:
+
+* Human
+* Mouse
+* Pig
+* Rat
+
+These variants' geometry and annotations are best viewed in the **Scaffold Creator** tool in the ABI Mapping Tools.
+On the web, the latest published generic stomach scaffold variants can be viewed on the
+`SPARC Portal <https://sparc.science/>`_ by searching for ``stomach``, filtering for models, selecting a variant and
+viewing the scaffold in its Gallery tab.
+
+The stomach scaffold script generates the scaffold mesh and geometry from an idealization of their shapes. The mesh is
+derived from ellipsoid and cubic functions based on a one dimensional central path with side axes controlling lateral
+dimensions. The parameters were carefully tuned for each species, and it is not recommended that these be edited.
+
+Instructions for editing the central path are given with the ABI Mapping Tools **Scaffold Creator** documentation.
+Note that the D2 and D3 derivatives control the side dimensions, and derivatives D12 and D13 control the rate of change
+of these along the central path. If editing, use the Interactive Functions to *Smooth derivatives*,
+*Make side derivatives normal* and *Smooth side cross derivatives* to make these as smooth as required.
+
+The mucosa, submucosa, circular muscle, longitudinal muscle and serosa layers of the stomach are fully represented on
+the scaffold when *Number of elements through wall* is set to ``4``. Alternatively, the entire stomach wall can be
+represented as a single layer by setting *Number of elements through wall* to ``1``.
+
+[A special ``Material`` parameter set is provided to allow new species' parameters to be developed from the material
+coordinates definition (see below).]
+
+Coordinates
+-----------
+
+The stomach scaffold defines the geometric, split and material coordinates.
+
+The geometric ``coordinates`` field gives an approximate, idealized representation of the stomach shape for the species,
+which is intended to be fitted to actual data for a specimen.
+
+The ``split coordinates`` follows the geometric field but provides continuous derivatives for nodes on the margin
```

Comment:
... provides disconnected and duplicated derivatives for nodes on the margin ...